pkg/security/resolvers/process/resolver.go

```diff
@@ -854,7 +854,7 @@ func (p *Resolver) SetProcessArgs(pce *model.ProcessCacheEntry) {
 }
 
 // GetProcessArgv returns the args of the event as an array
-func (p *Resolver) GetProcessArgv(pr *model.Process) ([]string, bool) {
+func GetProcessArgv(pr *model.Process) ([]string, bool) {
```

Comment:
I'm not sure I understand why we need to remove this function from the resolver 🤔